Source/WebCore/rendering/line/LineWidth.h

```diff
@@ -76,7 +77,7 @@ class LineWidth {
     bool fitsOnLineExcludingTrailingCollapsedWhitespace() const;
     void updateLineDimension(LayoutUnit newLineTop, LayoutUnit newLineWidth, float newLineLeft, float newLineRight);
 
-    RenderBlockFlow& m_block;
+    CheckedPtr<RenderBlockFlow> m_block;
```

Comment:
Since it used to be a reference, it should be a CheckedRef, not a CheckedPtr. Also, we should make it `const` so we don't need variables on the stack for it.